module/userProjects/helper.js

```diff
@@ -339,6 +341,8 @@ module.exports = class UserProjectsHelper {
                     }
                 }
 
+                await kafkaProducersHelper.pushImrovementProjectToKafka(projectUpdated);
```

Comment:
@rakeshSgr let the function name be "pushProjectForReporting"

Reply:
resolved